src/components/modals/FioCreateHandleModal.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react'
+import { Text, View } from 'react-native'
+import { AirshipBridge } from 'react-native-airship'
+import FastImage from 'react-native-fast-image'
+
+import { useHandler } from '../../hooks/useHandler'
+import s from '../../locales/strings'
+import { parseMarkedText } from '../../util/parseMarkedText'
+import { styled } from '../hoc/styled'
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { EdgeText } from '../themed/EdgeText'
+import { MainButton } from '../themed/MainButton'
+import { ThemedModal } from '../themed/ThemedModal'
+
+interface Props {
+  bridge: AirshipBridge<boolean>
+}
+
+export const FioCreateHandleModal = (props: Props) => {
+  const { bridge } = props
+
+  const theme = useTheme()
+  const styles = getStyles(theme)
+
+  const handleCancel = useHandler(() => {
+    bridge.resolve(false)
+  })
+
+  const handleConfirm = useHandler(() => {
+    bridge.resolve(true)
+  })
+
+  return (
+    <ThemedModal bridge={bridge} onCancel={handleCancel}>
+      <View style={styles.container}>
+        {/* TODO: Update placeholder image */}
+        <FastImage source={{ uri: 'https://content.edge.app/currencyIconsV3/fio/fio_dark.png' }} style={styles.icon} />
+        <GetFioHandleTitle>{parseMarkedText(s.strings.fio_free_handle_title_m)}</GetFioHandleTitle>
+        <EdgeText style={styles.message} numberOfLines={2} disableFontScaling>
+          {s.strings.fio_free_handle_congrats}
+        </EdgeText>
+        <EdgeText style={styles.message} numberOfLines={4} disableFontScaling>
```

Comment:
We need numberOflines for that exact reason of supporting longer strings. 

Without numberOfLines set, we get elipsis '...'

I'll increase numberOfLines further just in case